<ui version="4.0" >
 <class>StyleSheetDialog</class>
 <widget class="QDialog" name="StyleSheetDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Select Browser Style Sheet</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Style Sheet</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QRadioButton" name="defaultButton_" >
        <property name="text" >
         <string>Default</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" >
        <item row="0" column="0" colspan="2" >
         <widget class="QRadioButton" name="fileButton_" >
          <property name="text" >
           <string>From File</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLineEdit" name="pathEdit_" />
        </item>
        <item row="1" column="1" >
         <widget class="QPushButton" name="browseButton_" >
          <property name="text" >
           <string>&amp;Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton_" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton_" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton_</sender>
   <signal>clicked()</signal>
   <receiver>StyleSheetDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>251</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel" >
     <x>196</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton_</sender>
   <signal>clicked()</signal>
   <receiver>StyleSheetDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>325</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel" >
     <x>310</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileButton_</sender>
   <signal>toggled(bool)</signal>
   <receiver>pathEdit_</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>93</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel" >
     <x>95</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileButton_</sender>
   <signal>toggled(bool)</signal>
   <receiver>browseButton_</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>131</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel" >
     <x>320</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
